Lib/cProfile.py

```diff
@@ -139,7 +140,8 @@ def main():
         help="Save stats to <outfile>", default=None)
     parser.add_option('-s', '--sort', dest="sort",
         help="Sort order when printing to stdout, based on pstats.Stats class",
-        default=-1)
+        default=-1,
+        choices=list(pstats.Stats.sort_arg_dict_default.keys()))
```

Comment:
.keys() is not needed.

Reply:
By the way, *maybe* we should sort choices? Like sorted(...)?

```suggestion
        choices=sorted(pstats.Stats.sort_arg_dict_default)
```